crates/relayer-cli/src/commands/query/packet/util.rs

```diff
@@ -0,0 +1,63 @@
+use core::fmt;
+
+use serde::Serialize;
+
+use ibc_relayer::util::collate::{Collated, CollatedIterExt};
+use ibc_relayer_types::core::ics04_channel::packet::Sequence;
+use ibc_relayer_types::Height;
+
+pub use ibc_relayer::chain::counterparty::PendingPackets;
```

Comment:
I am not sure to understand why the `PendingPackets` is re-exported?

Reply:
Just as a convenience, this way one can get both the `PendingPackets` struct and its `Collated` counterpart with one import line.